src/Layout.js

```diff
@@ -0,0 +1,13 @@
+import ComponentRegistry from '../bower_components/metal/src/component/ComponentRegistry';
+import SoyComponent from '../bower_components/metal/src/soy/SoyComponent';
+import './Layout.soy';
+
+class Layout extends SoyComponent {
+    constructor(opt_config) {
```

Comment:
You don't have to override the constructor, since it's just calling the parent one. This was probably generated for you by an old version of generator-metal, but we don't use it anymore.